Adds new Level: Jarvis Gone Wrong (#17)
skills/secure-code-game: README.md

</details>
   

<details id=0>

> <summary><h2>Level X: Jarvis Gone Wrong</h2></summary>

Reviewer: Highlighting the level - where does this fit in the game flow?

Author: 👋  @jkcso, I will let you arrange the level number and `README.md` for the new season

_Welcome to "Secure Code Game"! :wave:_

### 📝 Storyline

> Jarvis, your trusty automation butler, has been acting up lately. Apparently, he's been experimenting with some "unofficial" pipelines that he found on the internet to [get the status page of GitHub](https://www.githubstatus.com/api/v2/status.json). While these actions promised to make your CI/CD pipeline faster and more efficient, they have also introduced some serious security vulnerabilities into your code. Do you have what it takes to fix the bug and progress to Level 1?

Reviewer: ```suggestion
Jarvis, your trusty automation butler, has been acting up lately. He's been experimenting with unofficial pipelines he found online to [get the GitHub status page](https://www.githubstatus.com/api/v2/status.json). While these actions promised to make your CI/CD pipeline faster and more efficient, they have also introduced serious security vulnerabilities into your code. Do you have what it takes to fix the bug and progress to Level 1?
```

### :keyboard: What's in the repo?

> You will find this file structure:

Reviewer: ```suggestion
For each level, you will find the same file structure:
```
Updating to stay consistent with the other levels

- `./github/workflows/jarvis.yml` includes the vulnerable GitHub Action workflow code to be reviewed
- `./github/workflows/jarvis-test.yml` tests if the vulnerable action is still present. If not, well done! 
- `hint.txt` offers a hint if you get stuck.
- `solution.yml` provides one working solution. There are several possible solutions.

### 🚦 Time to start!
1. Review the code in `./github/workflows/jarvis.yml`. Can you spot the bug?

> 1. Try to fix the bug. Ensure that you use or install zero dependencies to fix it.

Reviewer: ```suggestion
1. Try to fix the bug. Do not use or install any additional dependencies to fix it.
```
